sega_genesis_checksum_utility_for_python_3.py

```diff
@@ -142,12 +142,12 @@ def verify_console_name_from_header(open_file):
     True if the header of `open_file` contains the correct console name in the
     right location.
   """
-  CONSOLE_NAME_OFFSET = 0x100
-  open_file.seek(CONSOLE_NAME_OFFSET)
-  console_name = open_file.read(15)
+  memorybuffer = open_file.read()
+  memorybuffer[0x100:0x10c].decode('utf-8')
```

Comment:
Why are we calling `memorybuffer[0x100:0x10c].decode('utf-8')` twice? Once on line 146 and once on line 147. Can the line 146 just be removed entirely?

Reply:
Yeah, removed it entirely.